ingest/sqs/pools/transformer/pool_transformer.go

```diff
@@ -427,3 +473,58 @@ func (pi *poolTransformer) computeUSDCPoolLiquidityCapFromUOSMO(ctx sdk.Context,
 
 	return poolLiquidityCapUOSMO, noPoolLiquidityCapError
 }
+
+func (pi *poolTransformer) updateAlloyTrasmuterInfo(
+	ctx sdk.Context,
+	pool poolmanagertypes.PoolI,
+	cwPool cosmwasmpooltypes.CosmWasmExtension,
+	cosmWasmPoolModel *sqsdomain.CosmWasmPoolModel,
+	denoms *[]string,
+) error {
+	bz, err := pi.wasmKeeper.QuerySmart(ctx, cwPool.GetAddress(), []byte(`{"list_asset_configs":{}}`))
+	if err != nil {
+		return fmt.Errorf(
+			"error querying list_asset_configs for pool (%d) contrat_address (%s): %w",
+			pool.GetId(), pool.GetAddress(), err,
+		)
+	}
+	var assetConfigsResponse struct {
+		AssetConfigs []sqsdomain.TransmuterAssetConfig `json:"asset_configs"`
+	}
+
+	if err := json.Unmarshal(bz, &assetConfigsResponse); err != nil {
+		return fmt.Errorf(
+			"error unmarshalling asset_configs for pool (%d) contrat_address (%s): %w",
+			pool.GetId(), pool.GetAddress(), err,
+		)
+	}
+
+	bz, err = pi.wasmKeeper.QuerySmart(ctx, cwPool.GetAddress(), []byte(`{"get_share_denom":{}}`))
+	if err != nil {
+		return fmt.Errorf(
+			"error querying get_share_denom for pool (%d) contrat_address (%s): %w",
+			pool.GetId(), pool.GetAddress(), err,
+		)
+	}
```

Comment:
I wonder what is the impact of this on block processing time given that we are adding a few cosmwasm pool queries.

Would it be possible to do a benchmark on the in-place testnet?
A) No alloyed pools
B) 20 alloyed pools

For both, measure samples over 10 or so blocks and take the average.

While cumbersome, I think this would instill confidence and significantly derisk the upcoming release

Reply:
My internet was a bit broken, couldn't download the snapshot, so I did it with clean state. Let me know if you still want mainnet state.

measure time running pool transformer `Transform` function

`no pool avg  = 0.0000813042s`
`10 pools avg = 0.0074192627s`

Reply:
Looks reasonable to me. Thank you